sphinx/analysis.rst

```diff
@@ -0,0 +1,36 @@
+.. methods:
+
+Analysis Methods
+################
+
+Analysis Methods work on PlantData objects to produce high level analysis products, such as the long term AEP.
+These methods rely on the more generic toolkit modules, chaining them together to create reproducible analysis processes.
+
+Plant Level Analysis
+********************
+
+.. automodule:: openoa.analysis.aep
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. automodule:: openoa.analysis.eya_gap_analysis
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Turbine Level Analysis
+**********************
```

Comment:
I still view `TurbineLongTermGrossEnergy` as a plant-level analysis since it looks at the total power produced by all turbines in the plant. Maybe we save the "Turbine level analysis" label for methods that produce analysis metrics for individual turbines. But feel free to disagree with me here.

Reply:
Because this is two sections for four analysis methods (and only one method under one of the sections), I've removed the section titles for now, but we can revisit it later when we have more methods incorporated